activesupport/lib/active_support/testing/assertionless_tests.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module ActiveSupport
+  module Testing
+    # Allows to configure the behavior when the test case has no assertions in it.
+    # This is helpful in detecting broken tests that do not perform intended assertions.
+    module AssertionlessTests # :nodoc:
+      def self.prepended(klass)
+        klass.singleton_class.attr_reader :assertionless_tests_behavior
```

Comment:
I think we should make this a `class_attribute`, so that you can change the behavior on a per child class basis. (don't forget not pass `instance_accessor: false, instance_predicate: false`)

Reply:
Yeah, you can't just override the setter, best is to use `class_attirbute :_attr_name`, then from you override set `self._attr_name`.

Reply:
Thanks, that worked. Ready for review again.